<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourseEditDialog</class>
 <widget class="QDialog" name="CourseEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>课程编号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>480</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>课程名称</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_name">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>480</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>课程时间</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_time">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>学分</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_credit">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>139</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>540</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>学期数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_semester">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>139</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>540</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>教学地点</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_classroom">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>651</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>200</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_modify">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>250</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delete">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>300</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>480</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Genshin.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
